research-hub-web/src/app/components/equipment/equipment.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { AllEquipmentGQL, AllEquipmentQuery, AllSearchableContentPublicFieldsGQL, AllSearchableContentPublicFieldsQuery } from '../../graphql/schema';
+import { Observable } from 'rxjs';
+import { pluck, tap } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-equipment',
+  templateUrl: './equipment.component.html',
+  styleUrls: ['./equipment.component.scss']
+})
+export class EquipmentComponent implements OnInit {
+
+  public allEquipment$: Observable<AllEquipmentQuery['equipmentCollection']>;
+
+  constructor(public allEquipmentGQL: AllEquipmentGQL) { }
+
+  ngOnInit(): void {
+    try {
+      this.allEquipment$ = this.allEquipmentGQL.fetch()
+        .pipe(pluck('data', 'equipmentCollection'));
+    } catch (e) { console.log('Error loading equipment:', e); }
```

Comment:
console log that can be removed.

Reply:
Hmm, I don't mind this one personally - it's an actually handy `console.log`. If we ever get this error it means that the page hasn't been able to load at all so the user won't be shown anything anyway (things are seriously broken). What do you think? 🤔

Reply:
OK, no worries. I thought error logging might be useful for debugging, just thought I'd flag it :) Perhaps we should display an error to ask the user to reload.